<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSE Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 11pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelHostname">
      <property name="text">
       <string>HostName</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditHostname">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 172, 172);
color: rgb(170, 0, 0);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>hostname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>73</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="labelState">
      <property name="text">
       <string>State</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="lineEditState">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #3F3F3F;
color: #72D5A3;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Stop</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelIPv4">
      <property name="text">
       <string>IPv4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditIPv4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 172, 172);
color: rgb(170, 0, 0);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>73</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="labelRunningTime">
      <property name="text">
       <string>RuningTime</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="lineEditRunningTime">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #3F3F3F;
color: #72D5A3;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>00:00:00</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::VisualMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1025</number>
      </property>
      <property name="maximum">
       <number>49151</number>
      </property>
      <property name="value">
       <number>4355</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelPortRange">
      <property name="text">
       <string>(1025~49151)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Server Log</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditLog">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(63, 63, 63);
color: rgb(148, 191, 243);</string>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionStartOrStop"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionSQL"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartOrStop">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSQL">
   <property name="text">
    <string>SQL</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure Database</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
